src/layout/questionnaire-nav/components/arbo-simplifield.jsx

```diff
@@ -42,34 +42,37 @@ class ArboSimplified extends Component {
 
     return getSortedChildren(components, parent).map(key => {
       const subTree = renderComponentsByParent(components, key);
-      return (
-        <li
-          key={key}
-          className={isQuestion(components[key]) ? 'questions' : ''}
-        >
-          {components[key].children && components[key].children.length > 0 && (
-            <a
-              onClick={e => this.handleExpand(e, key)}
-              href="#"
-              aria-label="expand/collapse"
-            >
-              <span
-                className={`glyphicon ${
-                  this.state.expanded.indexOf(key) >= 0
-                    ? 'glyphicon-menu-down'
-                    : 'glyphicon-menu-right'
-                }`}
-              />
+      if(!isLoop(components[key])) {
+        return (
+          <li
+            key={key}
+            className={isQuestion(components[key]) ? 'questions' : ''}
+          >
+            {components[key].children && components[key].children.length > 0 && (
+              <a
+                onClick={e => this.handleExpand(e, key)}
+                href="#"
```

Comment:
a link with href=# is not a link. You should replace this element into a button .